docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -325,3 +325,44 @@ This issue might occur when you use a Zowe version of 1.12.0 or later. To resolv
 ```ZWED_node_https_certificateAuthorities="/path/to/zowe/keystore/local_ca/localca.cer-ebcdic","/path/to/carootcert.pem","/path/to/caintermediatecert.pem"```
  
 Recycle your Zowe server. You should be able to log in to the Zowe Desktop successfully now.
+
+### Browser unable to connect with a CIPHER error
```

Comment:
How about "unable to connect *due to* a CIPHER error"?

Reply:
Updated as you suggested